optional/fetchmail/fetchmail.py

```diff
@@ -64,7 +64,7 @@ def run(debug):
                 username=escape_rc_string(fetch["username"]),
                 password=escape_rc_string(fetch["password"]),
                 options=options,
-                folders=folders,
+                folders='' if fetch['protocol'].lower().startswith('pop') else folders,
```

Comment:
You can test for 'pop3', as this is defined very strictly:
`db.Column(db.Enum('imap', 'pop3'), nullable=False)`

Reply:
Ok, that's [008ff03](https://github.com/Mailu/Mailu/pull/2929/commits/008ff03882740ffedb260f3daa09e061c34e0ad4)